Source/Resources/Content/AadIam/DeviceSettings/Configuration.json

```diff
@@ -1,4 +1,5 @@
 {
+  "description":"What is it: Defines users allowed to join a device in Azure AD along with the security controls involved in process. \n Simeon Notes: This setting does not control Intune device enrollment, but users are required to have this permission to complete the Intune user-driven autopilot process. Simeon's baseline allows members of the Azure AD groups: 'BUILTIN - Device Enrollers' and 'BUILTIN - Microsoft 365 Users' to Azure AD join up to 100 devices per user, requiring MFA for each join. \n Learn More: https://docs.microsoft.com/en-us/azure/active-directory/devices/overview",
```

Comment:
**What does this do?**
Configures settings related to joining devices to Azure AD. The baseline only allows the groups 'Baseline - Device Enrollers' and 'Baseline - Microsoft 365 Users' to join up to 100 devices to Azure AD, and requires MFA to join.

**Why should you use this?**
To restrict the ability to join devices to Azure AD to only authorized groups.

**What is the end user impact?**
Only certain users may join devices and will be prompted for MFA to join.

**Learn more**
N/A